_data/pubs.yml

```
@@ -137,6 +137,16 @@
     - ai
     - hac
 
+- title: "FeedQUAC: Quick Unobtrusive Agent-Generated Commentary"
+  authors: ao Long, Kendra Wannamaker, Jo Vermeulen, George Fitzmaurice, Justin Matejka
+  conf: arXiv 2025
+  pub_date: 
```

Could you provide a rough time frame (it doesn't need to be completely accurate) to facilitate automatic sorting and display on the home page? As stated in the file header `pub_date: "YYYY-MM-DD"  # Only the year and month will be displayed, but you need to provide complete information for sorting purposes.`
```suggestion
  pub_date: "2025-04-23"
```